fierpy/fier.py

```diff
@@ -154,16 +162,23 @@ def match_dates(original: xr.DataArray, matching: xr.DataArray) -> xr.DataArray:
     # return the DataArray with only rows that match dates
     return original.where(original.time.isin(matching.time),drop=True)
 
-def find_fits(reof_ds: xr.Dataset, q_df: xr.DataArray, stack: xr.DataArray, train_size: float = 0.7, random_state: int = 0):
+def find_fits(reof_ds: xr.Dataset, q_df: xr.DataArray, stack: xr.DataArray, train_size: float = 0.7, random_state: int = 0, out_path='model_path'):
     """Function to fit multiple polynomial curves on different temporal modes and test results
 
     """
 
+    if not os.path.exists(out_path):
+       os.mkdir(out_path)
+    
     X = q_df
     y = reof_ds.temporal_modes
 
+    # ---- Randomly split data into 2 groups -----
     X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=train_size, random_state=random_state)
 
+    print(X_train)
```

Comment:
We should ideally convert print statements into [logging statements](https://docs.python.org/3/library/logging.html) for package code as we don't want to print a bunch of unneccessry info to users.